### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by KaiwalyaPataskar**
Using Active Storage feature, posters are added to Movies.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  # get 'movie/index'
```
Comment (anilmaurya) [inline config/routes.rb]: Remove commented routes

File: test/controllers/movie_controller_test.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class MovieControllerTest < ActionDispatch::IntegrationTest
+  test 'should get index' do
+    get movie_index_url
+    assert_response :success
```
Comment (anilmaurya) [inline test/controllers/movie_controller_test.rb:11]: Add proper test to verify Movies CRUD 
Also add test to verify poster upload

File: app/controllers/movies_controller.rb [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  def index
+    @movies = Movie.all
+  end
+
+  def new
+    @movie = Movie.new
+  end
+
+  def create
+    Movie.create!(movie_params)
```
Comment (rahulknojha) [inline app/controllers/movies_controller.rb]: What if the record is not created?

File: app/controllers/movies_controller.rb [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    redirect_to movies_path, notice: 'New Movie has been added successfully.'
+    # else
+    #  render 'new', notice: 'Hmm.. somthing went wrong!'
+    # end
+  end
+
+  def show
+    # Device gem for login
+    # Active storage gem for Upload image
```
Comment (rahulknojha) [inline app/controllers/movies_controller.rb]: ?????????????

File: app/controllers/movies_controller.rb [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    end
+  end
+
+  def find_movie
+    @movie = Movie.find(params[:id])
+  end
+
+  def destroy
+    @movie.destroy
+    redirect_to movies_path
```
Comment (rahulknojha) [inline app/controllers/movies_controller.rb]: Error handling?

File: app/helpers/movie_helper.rb [hunk 6]
```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+# MovieHelper module
+module MovieHelper
+end
```
Comment (rahulknojha) [inline app/helpers/movie_helper.rb:5]: If the file is blank then do not commit it.

File: app/views/movies/_form.html.erb [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+		<%= form.label(:budget) %>
+		<%= form.text_field :budget %>
+	</p>
+	</p>
+		<%= form.label(:poster) %>
+		<%= form.file_field :poster %>
+	</p>
+	<p>
+		<%= form.submit %>
+	</p>-->
```
Comment (rahulknojha) [inline app/views/movies/_form.html.erb]: Indentation ???

File: app/views/movies/index.html.erb [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+			    <%= image_tag ('default.jpg'), class: "img-responsive", width: 60, height: 80 %>
+				<% end %> &emsp;&emsp;
+				<%= link_to movie.title, movie_url(movie) %>
+			</td>
+			<td><br><%= movie.release_date.strftime('%d  %B  %Y') %></td>
+		</tr>
+	<% end %>
+	</table>
+</div>
+<br><br><br>
```
Comment (rahulknojha) [inline app/views/movies/index.html.erb:28]: Indentation?

File: app/views/movies/show.html.erb [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+			<%= link_to '<button class=btn-danger> DELETE </button>'.html_safe, movie_path(@movie), method: :delete, data:{confirm: 'Are you sure?'} %>
+		</div>
+		<div class=col-sm-4>
+			<% if @movie.poster.attached? %>
+				<%= image_tag @movie.poster, class: "img-responsive", width: 320, height: 375 %>
+			<% else %>
+			    <%= image_tag ('default.jpg'), class: "img-responsive", width: 320, height: 375 %>
+			<% end %>
+		</div>
+	</div>
```
Comment (rahulknojha) [inline app/views/movies/show.html.erb:29]: Indentation ??

File: app/views/movies/table.html.erb [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+			<% @movies.column_names.each do |column| %>
+				<td>
+					<%= movie.send(column) %>
+				</td>
+			<% end %>
+			<td><%= link_to "Edit", edit_movie_path(movie) %></td>
+			<td><%= link_to "Delete", movie_path(movie), method: :delete, data:{confirm: 
+				'Are you sure?'} %></td>
+		</tr>		
+	<% end %>
```
Comment (rahulknojha) [inline app/views/movies/table.html.erb:21]: indentation ?

File: config/routes.rb [hunk 11]
```diff
@@ -1,5 +1,11 @@
...
+  # get 'movie/new'
+  # get 'movie/show'
+  # get 'movie/edit'
+  # post 'create', to: 'movie#create'
```
Comment (rahulknojha) [inline config/routes.rb]: Remove commented lines if not used.

File: db/migrate/20190118075139_drop_table_film_personnel.rb [hunk 12]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+# Film_personnel...!  LOL
```
Comment (rahulknojha) [inline db/migrate/20190118075139_drop_table_film_personnel.rb]: LOL What is it ?, Give some meaningful comment.

**Comment by rahulknojha** [comment]
@KaiwalyaPataskar Please give some proper pull request name and branch name.

**Pull request merged**